tests/integration/test_infer.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+
+import requests
+
+from . import SERVER_URI
+
+
+class EndPointInfer(unittest.TestCase):
+    def test_segmentation(self):
+        model = "segmentation_left_atrium"
```

Comment:
lets have gpu check.. and skip if gpu is not enabled kind..
```
        if not torch.cuda.is_available():
            return
```

Reply:
why? this can happen on CPU even if it takes longer. it's just inference

Reply:
K.. some models don't work due to specific network/op not available on cpu.. we can merge if this run for both

Reply:
I see. OK. Changing